### PR TITLE
Added checks for input Tags.

### PR DESCRIPTION
**Pull request by koushdey**

### CODE REVIEW

File: Tasks/DockerV2/dockerbuild.ts [hunk 1]
```diff
@@ -42,7 +42,8 @@ export function run(connection: ContainerConnection, outputUpdate: (data: string
     let labelArguments = pipelineUtils.getDefaultLabels(addPipelineData);
 
     // get tags input
-    let tags = tl.getInput("tags").split(/[\n,]+/);
+    let tagInput = tl.getInput("tags");
```
Comment (ajinkya599) [inline Tasks/DockerV2/dockerbuild.ts:45]: `tagsInput`

**Comment by koushdey** [comment]
Will add test cases in another PR.

File: Tasks/DockerV2/dockerpush.ts [hunk 2]
```diff
@@ -20,7 +20,9 @@ function pushMultipleImages(connection: ContainerConnection, imageNames: string[
     // create chained promise of push commands
     if (imageNames && imageNames.length > 0) {
         imageNames.forEach(imageName => {
+            tl.debug(`KAI ${tags.length}`);
```
Comment (shigupt202) [inline Tasks/DockerV2/dockerpush.ts:23]: This is not needed, right?

File: Tasks/DockerV2/dockerpush.ts [hunk 3]
```diff
@@ -20,7 +20,9 @@ function pushMultipleImages(connection: ContainerConnection, imageNames: string[
...
             if (tags && tags.length > 0) {
+                tl.debug(`KAIii ${tags}`);
```
Comment (shigupt202) [inline Tasks/DockerV2/dockerpush.ts:25]: And this as well?

**Pull request merged**